packages/integration-test/src/stability.spec.ts

```diff
@@ -198,8 +312,11 @@ describe("stability check, assess a solution, reassess the solution, check all s
       "Miniblog.Core.sln"
     );
     await addSolution(app, solutionPath);
-    await app.stop();
-    app = await startApp();
-    return await runThroughSolution(solutionPath, "inplace");
+    await app.client.refresh();
+    const results = await runThroughSolution(solutionPath, "inplace");
```

Comment:
as we discussed, let's parameterize the target framework

Reply:
add here https://github.com/aws/porting-assistant-dotnet-ui/pull/42/commits/3bb1d45e38a815f8a4d0da47366b3568be1d95dc